assets/vsphere_problem_detector/12_prometheusrules.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: vsphere-problem-detector
+  namespace: openshift-cluster-storage-operator
+  labels:
+    role: alert-rules
+spec:
+  groups:
+    - name: vsphere-problem-detector.rules
+      rules:
+      - alert: VSphereOpenshiftNodeHealthFail
+        expr:  vsphere_node_check_errors == 1
+        for: 10m
+        labels:
+          severity: warning
+        annotations:
+          message: "Vsphere node health checks are failing on {{ $labels.node }} with {{ $labels.check }}"
```

Comment:
Is it OK to alert on each node separately? This may be quite spammy if all nodes are equally bad.

Reply:
@openshift/openshift-team-monitoring

Reply:
According to monitoring team:

> e.g you have 2 nodes for which vsphere_node_check_errors = 1, you'll receive 1 notification containing 2 alerts firing along with the message for each failing node 

I think it might be okay. It is not unusual for a 100 node cluster to all going wrong at once, but I think the entire class can be disabled at once and hence should be okay. If we remove node name from here, then it will be less useful I think because we can't tell from which node these alerts are coming.